src/portable/nuvoton/nuc505/dcd_nuc505.c

```diff
@@ -574,15 +579,6 @@ void USBD_IRQHandler(void)
       /* given ACK from host has happened, we can now set the address (if not already done) */
       if((USBD->FADDR != assigned_address) && (USBD->FADDR == 0)) USBD->FADDR = assigned_address;
```

Comment:
you could use the optional API `dcd_edpt0_status_complete()` to complete the address, although it causes the function to be invoked every control transfer. It is not a big of a deal since the number of control transfer is much smaller than normal endpoints (mostly in the enumeration).
https://github.com/hathach/tinyusb/blob/master/src/portable/microchip/samd/dcd_samd.c#L189